### ISSUE TITLE
Bulk `SecretsAPI.Resolve`

### ISSUE THREAD

**Issue by pgavlin** [1Password/onepassword-sdk-go#38]
It would be very nice to be able to resolve multiple secrets in bulk rather than resolving secrets one at a time. In the absence of a bulk API, we will likely need to parallelize the request we make in order to improve performance when resolving multiple secrets.

**Comment by Jackman3005** [comment]
I agree with this. We have 93 secrets to resolve on startup of our application. I am limiting it to 10 requests in parallel and it is still taking quite awhile to startup.

My project is written in _Typescript_ and I have created a similar [feature request in the onepassword-sdk-js repo](https://github.com/1Password/onepassword-sdk-js/issues/110). I offer some ideas for the API there as well. 

I wonder if the lack of this ability in these SDKs is more a reflection of the lack of this ability in the core product. Possibly not yet made available for service accounts?

**Comment by pgavlin** [comment]
Any updates on this? A bulk resolution API would be extremely beneficial for our use case.

**Comment by MOmarMiraj** [comment]
Hey @pgavlin, @Jackman3005 

With the release of the Go SDK v0.2.1, we have implemented a bulk secret reference function, you can use it via `Secrets().ResolveAll`. If you have feedback, feel free to open up another issue, will be closing this one!
